content/blog/summary-from-the-9th-annual-chapel-implementers-and-users-workshop-chiuw-2022.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Summary from The 9th Annual Chapel Implementers and Users Workshop (CHIUW 2022)
+date: 2022-07-15T19:31:06.645Z
+priority: ""
+author: Michelle Strout and Engin Kayraklioglu
+authorimage: /img/mstrout-headshot-small.jpg
+thumbnailimage: /img/engin.png
+tags:
+  - Chapel Programming Language
+  - GPU Programming
+  - High Performance Computing
+  - Open Source
+  - Arkouda (Data Analytics in Python powered by Chapel)
+---
+**Introduction**
+
+Programming today is complicated by the many kinds of parallelism that exist in everything from phones to laptops to supercomputers.  The open-source Chapel parallel programming language is being developed at HPE with the goals of making programming more productive / easier, while still enabling high performance that takes advantage of the wide variety of parallelism available today.
```

Comment:
Recommend rephrasing the start of this to:
"Programming today becomes complicated by the many kinds of..."

Recommend rephrasing the first part of the second sentence to:
"The open-source Chapel parallel programming language being developed at HPE aims at making parallel programming easier and more productive, taking advantage of the wide variety of parallelism available today to enable high performance."

Reply:
used most of this rewording, "The open-source Chapel parallel programming language makes parallel programming easier and more productive, while still enabling high performance that takes advantage of the wide variety of parallelism available today."